### ISSUE TITLE
Timer Entry ending the Next Day doesn't show the End Time

### ISSUE THREAD

**Issue by TR66** [xbmc/xbmc#19228]
### Describe the bug
Create a TV/Timer entry of type: Recording: One time (manual).  Set the Start date and End date to be on two consecutive days.  For example, Start date: Sunday, February 14, 2021; and End date: Monday, February 15, 2021.  Once you click OK to save the Timer entry, it will appear in the list of TV/Timers.  However, the End time will not be visible in the list.  It will be truncated and replaced by three dots (...).

## Expected Behavior
I would expect the entry to be fully displayed in the list of TV/Timers, which should include the End time.

## Actual Behavior
The End time is not visible in the list of TV/Timers.  It is truncated and replaced by three dots (...).

## Possible Fix
Increase the width of the TV/Timers list; so that it is no longer necessary to truncate the End time.

### To Reproduce
Create a TV/Timer entry of type: Recording: One time (manual).  Set the Start date and End date to be on two consecutive days.  For example, Start date: Sunday, February 14, 2021; and End date: Monday, February 15, 2021.  Once you click OK to save the Timer entry, it will appear in the list of TV/Timers.  However, the End time will not be visible in the list.  It will be truncated and replaced by three dots (...).

### Debuglog
The debuglog can be found here: https://paste.kodi.tv/uxowubihuv.kodi

### Screenshots 
Here are some links or screenshots to help explain the problem:
![Kodi Timer Entry Cut-off](https://user-images.githubusercontent.com/65628919/107870930-52313280-6e5a-11eb-8f26-3adfec563438.PNG)

### Your Environment
Used Operating system:
 - [ ] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [x] Windows
 - [ ] Windows UWP

 - Operating system version/name:  Windows 10 Pro (64-bit) Version 1909, Build 18363.1316
 - Kodi version:  19.0-BETA2 (18.9.821) Git:20201207-8cc9e80e41

**Comment by joseluismarti** [comment]
Hi @TR66, I've not been able to reproduce the bug in v19, the full text is visible.  

Here's a screenshot:
![timer](https://user-images.githubusercontent.com/76552691/108833720-85e42900-75cd-11eb-9f17-5f9bce042533.png)

**Comment by TR66** [comment]
Hi @joseluismarti, Thanks for checking into my report here.  I checked this again myself and (embarrassingly) I'm finding that I can't reproduce the problem either.  I'm not sure what could have changed in the last week or so.  I tested for this on the exact same system; and the system has had no new updates since my initial report.  Yet now everything in the Timer list displays correctly.  I know it wasn't my imagination; since I have the evidence in my screenshot.  But since the problem is no longer reproducible, all that makes sense for now is to just close out this report.  Thanks again for checking though.  I do appreciate you looking into it and replying here.  I'm closing this issue for now.
